qmd/_quarto.yml

```diff
@@ -4,26 +4,38 @@ project:
 
 book:
   title: "Public Health Environmental Surveillance Open Data Model (PHES-ODM) Documentation"
-  subtitle: "Version 2.0.0"
+  output-file: "ODM-documentation-v2.0.1"
+  version: "2.0.1"
+  subtitle: |
+      Dictionary v2.0.0. <br>Documentation V2.0.1.
```

Comment:
<img width="735" alt="image" src="https://user-images.githubusercontent.com/16597423/236511749-3ef0f771-9f7b-4ef8-91da-2859480e5d0b.png">
Using the br between versions looks strange to me. I believe keeping them on the same line would be better.